contracts/feature/Ownable.sol

```diff
@@ -11,7 +11,20 @@ import "./interface/IOwnable.sol";
 
 abstract contract Ownable is IOwnable {
     /// @dev Owner of the contract (purpose: OpenSea compatibility)
-    address public override owner;
+    address private _owner;
+
+    /// @dev Reverts if caller is not the owner.
+    modifier onlyOwner() {
+        if (msg.sender != _owner) {
+            revert Ownable__NotAuthorized();
+        }
+        _;
+    }
+
+    /// @dev Returns the owner of the contract.
+    function owner() public view returns (address) {
```

Comment:
if i remember correctly, `owner` is a standard that things like Opensea fetches, i think this should be fine, but we should make sure its still compatible? @jakeloo @nkrishang do you know?